lib/plugins/aws/custom-resources/index.js

```diff
@@ -179,6 +179,19 @@ async function addCustomResourceToService(awsProvider, resourceName, iamRoleStat
   };
   Resources[customResourceFunctionLogicalId] = customResourceFunction;
 
+  // configure vpc for the custom resource function
```

Comment:
Let's remove this comment (it describes what code does, and doesn't add any new information)

Reply:
This comment remains not addressed

Reply:
removed the comment and updated the PR.